include/crud-posts.php

```diff
@@ -324,4 +325,38 @@ public function create_media_translation( $post_id, $lang ) {
 		do_action( 'pll_translate_media', $post_id, $tr_id, $lang->slug );
 		return $tr_id;
 	}
+
+	/**
+	 * Forces tags translation when a post is updated, due to `tags_input` paramter being removed in `wp_update_post()`.
+	 *
+	 * @since 3.4.4
+	 *
+	 * @param int     $post_id      Post ID, unused.
+	 * @param WP_Post $post_after   Post object following the update.
+	 * @param WP_Post $post_before  Post object before the update.
+	 * @return void
+	 */
+	public function force_tags_translation( $post_id, $post_after, $post_before ) {
+		static $avoid_recursion = false;
+
+		if ( $avoid_recursion ) {
+			return;
+		}
+
+		$avoid_recursion = true;
+
+		$post_before = $post_before->to_array();
+
+		if ( empty( $post_before['tags_input'] ) ) {
+			$avoid_recursion = false;
+			return;
+		}
+
+		$post_after               = $post_after->to_array();
+		$post_after['tags_input'] = $post_before['tags_input'];  // Will force `PLL_CRUD_Posts::set_object_terms()` to do its job.
+
+		wp_insert_post( $post_after );
+
+		$avoid_recursion = false;
```

Comment:
I'm looking for a way not to execute the complete `wp_insert_post()` a second time.
Would this work?
```suggestion
		if ( ! empty( $post_before['tags_input'] ) ) {
			// Let  ensure that  `PLL_CRUD_Posts::set_object_terms()` will do its job.
			wp_set_post_tags( $post_id, $post_before['tags_input'] ) );
		}
```

Reply:
Yeah! It works! Done in 174e49ee 🚀